gramps/plugins/db/dbapi/postgresql.py

```diff
@@ -94,15 +94,10 @@ def commit(self):
     def rollback(self):
         self.connection.rollback()
 
-    def try_execute(self, sql):
-        sql = self._hack_query(sql)
-        sql = sql.replace("BLOB", "bytea")
-        try:
-            self.cursor.execute(sql)
-        except Exception as exc:
-            self.cursor.execute("rollback")
-            #print("ERROR:", sql)
-            #print(str(exc))
+    def table_exists(self, table):
+        self.cursor.execute("SELECT COUNT(*) FROM information_schema.tables "
+                            "WHERE table_name='%s';" % table)
```

Comment:
Don't use % on sql statements; use ? and follow with args (see other examples of execute). Otherwise, you can end up with bad errors and SQL injection hacks.